projects/CNSPRCY/services/cnsprcy/module.nix

```diff
@@ -0,0 +1,70 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.cnsprcy;
+  libDir = "/var/lib/cnsprcy";
```

Comment:
This could also be an option, maybe. Although this would usually be called [stateDir](https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#RuntimeDirectory=).